tusk_11py.3.py

```diff
@@ -0,0 +1,14 @@
+class CustomContextManager:
+    def __enter__(self):
+        print("=" * 10)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("=" * 10)
+        return self
+
+    def exit(self, ex_type, ex_value, traceback):
```

Comment:
To reach the goal of this task you need to implement 2 methods: `__enter__` and `__exit__`. Method `exit` does nothing here and it will be never called if you work with this class as with a ContextManager.